src/frontend/qt_sdl/main.cpp

```diff
@@ -3376,7 +3509,20 @@ int main(int argc, char** argv)
 
     QObject::connect(&melon, &QApplication::applicationStateChanged, mainWindow, &MainWindow::onAppStateChanged);
 
-    mainWindow->preloadROMs(options->dsRomPath, options->gbaRomPath, options->boot);
+    const auto prepareRomPath = [&](const std::optional<QString>& romPath, const std::optional<QString>& romArchivePath) -> QStringList {
```

Comment:
{ on the next line

Reply:
Fixed in 04061eda0b08c0f51428fb4dc93cc2f7e8f37686.